packages/yourloops/test/integration/pages/care-team-settings/care-team.hcp.spec.tsx

```diff
@@ -87,4 +92,22 @@ describe('HCP care team settings page', () => {
 
     await testCareTeamLayout()
   })
+
+  it('should be able to update monitoring alerts parameters when user is admin of the team', async () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏We could also add test cases to check that both HCP admin and non-admin can edit monitoring alert parameters at the patient level?